### PR TITLE
cephfs-shell: modify and add descriptions for commands

### PR DESCRIPTION
**Pull request by tengjie5**
Fixes: https://tracker.ceph.com/issues/62074
Signed-off-by: jie teng tengjie5@asiainfo.com

## Contribution Guidelines
- To sign and title your commits, please refer to [Submitting Patches to Ceph](https://github.com/ceph/ceph/blob/main/SubmittingPatches.rst).

- If you are submitting a fix for a stable branch (e.g. "pacific"), please refer to [Submitting Patches to Ceph - Backports](https://github.com/ceph/ceph/blob/master/SubmittingPatches-backports.rst) for the proper workflow.

## Checklist
- Tracker (select at least one)
  - [x] References tracker ticket
  - [ ] Very recent bug; references commit where it was introduced
  - [ ] New feature (ticket optional)
  - [ ] Doc update (no ticket needed)
  - [ ] Code cleanup (no ticket needed)
- Component impact
  - [ ] Affects [Dashboard](https://tracker.ceph.com/projects/dashboard/issues/new), opened tracker ticket
  - [ ] Affects [Orchestrator](https://tracker.ceph.com/projects/orchestrator/issues/new), opened tracker ticket
  - [x] No impact that needs to be tracked
- Documentation (select at least one)
  - [ ] Updates relevant documentation
  - [x] No doc update is appropriate
- Tests (select at least one)
  - [ ] Includes [unit test(s)](https://docs.ceph.com/en/latest/dev/developer_guide/tests-unit-tests/)
  - [ ] Includes [integration test(s)](https://docs.ceph.com/en/latest/dev/developer_guide/testing_integration_tests/)
  - [ ] Includes bug reproducer
  - [x] No tests

<details>
<summary>Show available Jenkins commands</summary>

- `jenkins retest this please`
- `jenkins test classic perf`
- `jenkins test crimson perf`
- `jenkins test signed`
- `jenkins test make check`
- `jenkins test make check arm64`
- `jenkins test submodules`
- `jenkins test dashboard`
- `jenkins test dashboard cephadm`
- `jenkins test api`
- `jenkins test docs`
- `jenkins render docs`
- `jenkins test ceph-volume all`
- `jenkins test ceph-volume tox`
- `jenkins test windows`
</details>

### CODE REVIEW

**Comment by vshankar** [comment]
@neesingh-rh PTAL

**Review by neesingh-rh — COMMENTED** [review]
If we are taking care of descriptions, I guess `complete_mv` also need modification.

**Comment by tengjie5** [comment]
> 
@neesingh-rh  hi，description of mv has modified，please review again.

**Comment by neesingh-rh** [comment]
Pls combine the change in single commit, no need to have a new one for suggested change.

**Comment by neesingh-rh** [comment]
make check is failing for :
```
py3: install_deps /home/jenkins-build/build/workspace/ceph-pull-requests/src/tools/cephfs/shell> python -I -m pip install flake8
py3: freeze /home/jenkins-build/build/workspace/ceph-pull-requests/src/tools/cephfs/shell> python -m pip freeze --all
py3: flake8==6.1.0,mccabe==0.7.0,pip==22.3.1,pycodestyle==2.11.0,pyflakes==3.1.0,setuptools==65.6.3,wheel==0.38.4
py3: commands[0] /home/jenkins-build/build/workspace/ceph-pull-requests/src/tools/cephfs/shell> flake8 --ignore=W503 --max-line-length=100 cephfs-shell
cephfs-shell:1125:101: E501 line too long (120 > 100 characters)
py3: exit 1 (1.09 seconds) /home/jenkins-build/build/workspace/ceph-pull-requests/src/tools/cephfs/shell> flake8 --ignore=W503 --max-line-length=100 cephfs-shell pid=436003
  py3: FAIL code 1 (10.59=setup[9.50]+cmd[1.09] seconds)
  evaluation failed :( (10.88 seconds)
```

**Comment by neesingh-rh** [comment]
Your commit(s) misses Signed-off-by, Pls add.

**Comment by dparmar18** [comment]
and please remove the redundant commits

File: src/tools/cephfs/shell/cephfs-shell [hunk 1]
```diff
@@ -1122,7 +1122,7 @@ class CephFSShell(Cmd):
         """
         return self.complete_filenames(text, line, begidx, endidx)
 
-    mv_parser = argparse.ArgumentParser(description='Move File.')
+    mv_parser = argparse.ArgumentParser(description='Rename a file or Move a file from source path to the destination.')
```
Comment (dparmar18) [inline src/tools/cephfs/shell/cephfs-shell:1125]: break this line into two

**Pull request closed without merging**